src/index.spec.ts

```diff
@@ -4,9 +4,149 @@ import {
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
+/*test('Testing time!', () => {
   console.log('welcome');
+});*/
+
+describe('isInteger()',() => {
+  it("should return true if value is a number", () => {
+    expect(isInteger(1)).toBe(true);
+    expect(isInteger(-1)).toBe(true);
+    expect(isInteger(0)).toBe(true);
+  });
+  it('should return false if value is a string', () => {
+    expect(isInteger('test')).toBe(false);
+    expect(isInteger('111')).toBe(false);
+    expect(isInteger('-123')).toBe(false);
+  });
+});
+
+describe('toLowerCase()', () => {
+  it('should return a default message if not pass a string', () => {
+    const defaultMessage = 'Please provide a string';
+    expect(toLowerCase('')).toBe(defaultMessage);
+    expect(toLowerCase(null)).toBe(defaultMessage);
+    expect(toLowerCase(undefined)).toBe(defaultMessage);
+  });
+  it('should return a string in lower case', () => {
+    expect(toLowerCase('TEST')).toBe('test');
+    expect(toLowerCase('test')).toBe('test');
+    expect(toLowerCase('TeSt')).toBe('test');
+  });
+});
+
+describe('removeDuplicatesFromArray()', () => {
+  it('should throw an error if not array provided', () => {
+    const errorMessage = 'please provide an array of numbers or strings';
+    expect(() => removeDuplicatesFromArray(null)).toThrowError(errorMessage);
+    expect(() => removeDuplicatesFromArray(undefined)).toThrowError(errorMessage);
+    expect(() => removeDuplicatesFromArray({} as any)).toThrowError(errorMessage);
+  });
+  it('should return an array with no duplicates', () => {
+    const oneElement = [1];
+    expect(removeDuplicatesFromArray(oneElement)).toEqual([1]);
+
+    const numbers = [1, 2, 3, 4, 5, 6];
+    expect(removeDuplicatesFromArray(numbers)).toEqual([1, 2, 3, 4, 5, 6]);
+
+    const repeatedNumbers = [1, 1, 1, 1, 1, 1];
+    expect(removeDuplicatesFromArray(repeatedNumbers)).toEqual([1]);
+
+    const strings = ['a', 'b', 'c', 'd', 'e', 'f'];
+    expect(removeDuplicatesFromArray(strings)).toEqual(['a', 'b', 'c', 'd', 'e', 'f']);
+
+    const repeatedStrings = ['a', 'a', 'a', 'a', 'a', 'a'];
+    expect(removeDuplicatesFromArray(repeatedStrings)).toEqual(['a']);
+
+    const stringAndNumbers = [1, 'b', 'c', 4, 'e', 6];
+    expect(removeDuplicatesFromArray(stringAndNumbers)).toEqual([1, 'b', 'c', 4, 'e', 6]);
+
+    const repeatedStringAndNumbers = ['a', 'a', 1, 1, null, undefined];
+    expect(removeDuplicatesFromArray(repeatedStringAndNumbers)).toEqual(['a', 1, null, undefined]);
+  })
```

Comment:
The testing is pretty good. But I think you can use AAA to keep the test case ordered and clean. 

AAA is an acronym for arrange, act and assert. [It's a pattern](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) for writing test cases.

In summary, you should define and create all you required inputs at first(arrange), after you should call and invoke all the logic being tested(act), and finally compare your results with the expected output(assert).